### DISCUSSION TITLE
Do you have umlauts

### DISCUSSION THREAD

**Discussion by sdf121234** [jonaswinkler/paperless-ng#1012] [Support]
Hello.  May i ask the german speaking users, if you have umlauts in content of the documents, which are recognized by paperless ? In my case, i only have u instead of ü (or a = ä /o = ö). 

Thank you very much.

**Comment by Burschi500** [comment]
Yes, although i do most of the scanning with mobile and app. Maybe install tesseract-ocr-deu and set PAPERLESS_OCR_LANGUAGE to the language most of your documents are written in.

**Reply by sdf121234** [reply]
Ok, thank you very much. I will try to figure it out.

**Reply by sdf121234** [reply]
I solved the problem,. I am using ppls as a docker file on synology. I added the line **PAPERLESS_OCR_LANGUAGE** with **deu** as a variable. Even if i also set PAPERLESS_OCR_LANGUAGES with deu before, now it runs. also with the umlauts.  thank you very much.
